samples/CookieSharing/README.md

```diff
@@ -0,0 +1,9 @@
+# Cookie Sharing Sample App
+
+The sample illustrates cookie sharing across three apps that use cookie authentication:
+
+* ASP.NET Core 2.0 Razor Pages app without using ASP.NET Core Identity
```

Comment:
I'd define the exact final project names here and include these long form description of what they are